packages/components/src/components/calendar/views/calendar-days.jsx

```diff
@@ -124,7 +124,10 @@ const getDays = ({
                         classNameTarget='dc-calendar__cell-tooltip'
                         classNameTargetIcon='dc-calendar__cell-tooltip-icon'
                         icon='dot'
+                        is_bubble_hover_enabled
                         message={message}
+                        zIndex={9999}
+                        should_activate_cursor
```

Comment:
`activate` doesn't sound right. Maybe we can change it to `should_show_cursor`.

Reply:
@balakrishna-binary, ok

Reply:
Fixed